app/assets/stylesheets/_addtocalendar.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * AddToCalendar Base CSS
+ * http://addtocalendar.com
+ */
+
+/* Base */
+
+.addtocalendar var{
+    display: none;
+}
+
+.addtocalendar {
+    position: relative;
+    display: inline-block;
+    background: transparent!important;
+}
+
+.atcb-link {
+    display: block;
+    outline: none!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces